File: 2023_04_05/8958.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();
        String[] test = new String[num];
        int sum;

        for(int i = 0; i < num; i++) {
            test[i] = scanner.next();
        }

        for(int i = 0; i < num; i++) {
            sum = 0;
            int score = 0;
            for(int j = 0; j < test[i].length(); j++) {
                if(test[i].charAt(j) == 'O'){
                    score++;
                    sum += score;
                } else {
                    score = 0;
                }
            }

            System.out.println(sum);
        }

        scanner.close();
    }
}

File: 날짜별 (2023-2024.7)/2024/2024_07_06/8958.py
n = int(input())

for _ in range(n):
  score = input()
  sum = 0
  count = 0

  for i in range(len(score)):
    if score[i] == 'O':
      count += 1
      sum += count
    else:
      count = 0
  
  print(sum)